experimental/Schemes/CoveredScheme.jl

```diff
@@ -615,6 +615,15 @@ function coverings(X::AbsCoveredScheme) ::Vector{<:Covering}
   return coverings(underlying_scheme(X))
 end
 
+@Markdown.doc """
+    default_covering(X::AbsCoveredScheme)::Covering
+
+Returns the default covering for ``X``.
+"""
+function default_covering(X::AbsCoveredScheme)::Covering
+  return default_covering(underlying_scheme(X))
```

Comment:
Do you perhaps mean
```suggestion
function default_covering(X::AbsCoveredScheme)
  return default_covering(underlying_scheme(X))::Covering
```
The difference is that your current code inserts a call to `convert(Covering, ret)` where `ret` is the return value; while the suggested change just inserts a type assertion, which verifies that the return value is of type `Covering` and otherwise throws an error

Reply:
I didn't know that. Thanks.